tests/jupyterhub_test.py

```diff
@@ -16,11 +16,33 @@ def setup_module(module):
 
 
 def test_check_auth_jupyterhub(fc):
+    import sirepo.srdb
+    from pykern import pkio
+    from pykern.pkunit import pkeq
+
+    def _get_num_jupyterhub_user_dirs():
+        return len(pkio.sorted_glob(sirepo.srdb.root().join("jupyterhub/user/**"))) - 1
+
+    def _jupyter_user_dir_exists(uid):
+        for x in [
+            os.path.basename(d)
+            for d in pkio.sorted_glob(sirepo.srdb.root().join("jupyterhub/user/**"))
+        ]:
+            if uid.lower() in x:
+                return True
+        return False
+
+    fc.sr_logout()
+    n = _get_num_jupyterhub_user_dirs()
+    fc.sr_get("checkAuthJupyterHub")
+    pkeq(n, _get_num_jupyterhub_user_dirs())
     fc.sr_login_as_guest()
-    # user doesn't exist
+    pkeq(False, _jupyter_user_dir_exists(fc.sr_uid))
```

Comment:
It's generally [not good style](https://docs.quantifiedcode.com/python-anti-patterns/readability/comparison_to_true.html) to test for equals False or True.
`pkok(not _jupyter_user_dir_exists(fc.sr_uid))`

Reply:
[e474deb](https://github.com/radiasoft/sirepo/pull/6921/commits/e474debba2686e7e319531ca53708db538f023f7)